src/components/FeatureProjects.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import styled from 'styled-components';
+import { Wrapper, SectionHeader } from 'components/GlobalStyles'
+import ProjectList from 'components/ProjectList';
+
+export const FeatureProjects = () => {
+  return (
+    <FeatureWrapper primary>
+      <SectionHeader>FEATURE PROJECTS</SectionHeader>
+      <ProjectList />
```

Comment:
I think it's really nice that you kept this component "clean" and import the projectList